.editorconfig

```diff
@@ -3,9 +3,11 @@ root = true
 [*]
 indent_style = space
 indent_size = 2
+tab_width = 2
 end_of_line = lf
 charset = utf-8
 trim_trailing_whitespace = true
+max_line_length = 120
```

Comment:
May worth check existing effective Polkadot editor/eslint/prettier defined and effective in cennznet/api.js, which I suppose it is `80`, need to be confirmed.